### PR TITLE
fix(app): modal tweaks

### PR DESCRIPTION
**Pull request by ewagoner**
# Overview

This PR addresses round one of DQA feedback for the modal component.

Closes RAUT-432

# Test Plan

Manually verified modal placement and content spacing on both ODD and simulated app.

# Changelog

- Centered modal component on ODD screen
- Adjusted padding and typography for run cancellation model

# Review requests

Verify modals are now centered. Verify both varieties of run cancellation modals now match designs.

# Risk assessment

Low

### CODE REVIEW

**Comment by ewagoner** [comment]
The check failure here doesn't seem real. It passes for me locally and the error itself doesn't make sense with the code it's testing.

**Comment by ewagoner** [comment]
> The check failure here doesn't seem real. 

Turned out to be very real. Jethary fixed the same thing yesterday on a different ticket, and it was an undetected merge conflict.

File: app/src/organisms/OnDeviceDisplay/RunningProtocol/ConfirmCancelRunModal.tsx [hunk 1]
```diff
@@ -83,23 +82,12 @@ export function ConfirmCancelRunModal({
       <Flex flexDirection={DIRECTION_COLUMN}>
         <Flex
           flexDirection={DIRECTION_COLUMN}
-          padding={SPACING.spacing32}
           gridGap={SPACING.spacing12}
+          paddingBottom={SPACING.spacing40}
+          paddingTop={`${isActiveRun ? SPACING.spacing40 : '0'}`}
```
Comment (koji) [inline app/src/organisms/OnDeviceDisplay/RunningProtocol/ConfirmCancelRunModal.tsx]: Why do we need 40px?
Screenshot 2023-05-17 at 12 53 43 PM

Comment (ewagoner) [reply app/src/organisms/OnDeviceDisplay/RunningProtocol/ConfirmCancelRunModal.tsx]: Good catch. I meant to change that to 32.

Comment (ewagoner) [reply app/src/organisms/OnDeviceDisplay/RunningProtocol/ConfirmCancelRunModal.tsx]: Fixed.

**Review by jerader — APPROVED** [review]
🦖

**Pull request merged**